docs/howtos/DataPlane Generation - DPG/05union.mdx

```diff
@@ -14,6 +14,11 @@ At a glance, JS and Python supports natively union, while Java and C# will use o
 <TabItem value="typespec" label="TypeSpec" default>
 
 ```typespec
+@service({
```

Comment:
Will adding this impact the code for other SDKs?

Reply:
It just define a service and namespace, it will not affected the Union behavior.